crates/sui-types/src/messages.rs

```diff
@@ -1098,15 +1172,49 @@ impl TransactionData {
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, Hash)]
 pub struct SenderSignedData {
     pub intent_message: IntentMessage<TransactionData>,
-    pub tx_signature: GenericSignature,
+    pub tx_signatures: Vec<GenericSignature>,
```

Comment:
please add a comment explaining what sort of things can appear in this list.

Reply:
done